### PR TITLE
[new] Add paver@1.3.3 w/ git auto-update

### LINKED ISSUE #10298: [Request] Add paver
**Library name:** paver
**Git repository url:** https://github.com/terrymun/paver
**License(s):** MIT

### PR DESCRIPTION
**Pull request by VPashkov**
Pull request for issue: #10298

# Profile of the lib
 * Git repository (required): https://github.com/terrymun/paver
 * NPM package url (optional): https://www.npmjs.com/package/@terrymun/paver
 * License and its reference: [MIT](https://github.com/terrymun/paver/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 62
   - Count of watchers: 6
   - Count of forks: 21

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by VPashkov** [comment]
@PeterDaveHello, I'll try to setup git auto-update instead of npm.

**Comment by PeterDaveHello** [comment]
@VPashkov Thanks

File: ajax/libs/paver/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "paver",
+  "filename": "js/jquery.paver.min.js",
+  "version": "1.3.3",
+  "description": "A minimal panorama/image viewer replicating the effect seen in Facebook Pages app.",
+  "homepage": "http://terrymun.github.io/paver/",
```
Comment (pvnr0082t) [inline ajax/libs/paver/package.json]: Let's use `https` instead of `http` for github.io domain. Thanks.

File: ajax/libs/paver/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "git",
+    "url": "https://github.com/terrymun/paver/"
+  },
+  "keywords": [
+    "paver",
+    "panorama",
+    "photo"
+  ],
+  "license": "MIT",
+  "dependencies": {
```
Comment (pvnr0082t) [inline ajax/libs/paver/package.json]: We don't need `"dependencies"`. Let's remove this.

File: ajax/libs/paver/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "**/*.js"
```
Comment (pvnr0082t) [inline ajax/libs/paver/package.json]: @VPashkov I think it's more flexible to use `"js/*paver*"`.

Comment (PeterDaveHello) [inline ajax/libs/paver/package.json]: it's also a little bit not safe or no sense to use `**/` under a non-dist folder

**Comment by PeterDaveHello** [comment]
@pvnr0082t please help squash and rebase the branch with the latest master branch.

**Comment by PeterDaveHello** [comment]
ping @kennynaoh

File: ajax/libs/paver/package.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  "description": "A minimal panorama/image viewer replicating the effect seen in Facebook Pages app.",
+  "homepage": "https://terrymun.github.io/paver/",
+  "author": {
+    "name": "Terry Mun",
+    "email": "hello@terrymun.com",
+    "homagepage": "http://terrymun.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/terrymun/paver/"
```
Comment (kennynaoh) [inline ajax/libs/paver/package.json]: I think it's better to use `https://github.com/terrymun/paver.git` here, Thanks!

File: ajax/libs/paver/package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "paver",
+  "filename": "js/jquery.paver.min.js",
+  "version": "1.3.3",
+  "description": "A minimal panorama/image viewer replicating the effect seen in Facebook Pages app.",
+  "homepage": "https://terrymun.github.io/paver/",
```
Comment (kennynaoh) [inline ajax/libs/paver/package.json]: @PeterDaveHello The homepage will redirect the url to `https://terrymun.github.io/paver/demo/index.html`. Is it necessary to change the url?

File: ajax/libs/paver/package.json (hunk 4 above)
Comment (PeterDaveHello) [inline ajax/libs/paver/package.json]: Nope, it hurts nothing, unless you can't clone from that url.

File: ajax/libs/paver/package.json (hunk 5 above)
Comment (PeterDaveHello) [inline ajax/libs/paver/package.json]: Nope, it's fine.

**Comment by VPashkov** [comment]
@PeterDaveHello, @pvnr0082t, @kennynaoh many thanks for review!

**Pull request merged**